webapp/tsconfig.base.json

```diff
@@ -11,6 +11,7 @@
     "importHelpers": true,
     "target": "es2020",
     "module": "es2020",
+    "skipLibCheck": true,
```

Comment:
I had to put this in because I started getting TS compile errors coming from PouchDB types here in webapp. I am not exactly sure what changed so that this started being a problem.  :thinking:   Perhaps is related to now Pouch types are also transitive deps from `@medic/cht-datasource`....  :shrug:   Either way, I guess we don't really need to be validating the types for deps anyway...